### DISCUSSION TITLE
注释套注释问题

### DISCUSSION THREAD

**Discussion by PirateBook1** [banned-historical-archives/banned-historical-archives.github.io#1867] [Q&A] [answered]
<img width="232" alt="截图" src="https://user-images.githubusercontent.com/113572850/213097910-09b79a39-0c5b-470a-9cec-d5e72f12027f.png">
@banned-historical-archives 这个问题挺普遍的，得尽快解决了

**Comment by PirateBook1** [comment, marked as answer]
或者讨论一下解决办法

**Reply by banned-historical-archives** [reply]
就像原文一样放在括号里吧

**Reply by PirateBook1** [reply]
@banned-historical-archives 
> （恩格斯在1888年英文版上加的注。）〔2〕

在注释里也像原文一样加括号吗，序号按照上一条的往下排

**Reply by banned-historical-archives** [reply]
这个2是什么意思？

**Reply by PirateBook1** [reply]
@banned-historical-archives 
![image](https://user-images.githubusercontent.com/113572850/213154356-bce5a6a8-8378-4e4d-8961-5a12bb8b4e7c.png)
![image](https://user-images.githubusercontent.com/113572850/213154440-0a40757f-7c89-47c0-979e-adb93b5d7956.png)
![image](https://user-images.githubusercontent.com/113572850/213154535-ab75ad22-2e23-40e0-94f8-e1315e4dcb23.png)
目前是像在原文里加〔〕一样在注释里也加〔〕吗

**Reply by PirateBook1** [reply]
> 这个2是什么意思？

如果注释里也可以用〔〕就没什么问题了

**Reply by banned-historical-archives** [reply]
恩 也一样加〔〕，只是没有跳转的效果

**Comment by banned-historical-archives** [comment]
没看懂

**Reply by PirateBook1** [reply]
“至今所有一切社会的历史”，这句话有一个恩格斯写的脚注（原文），针对这个脚注，后人（编者）又做了一个尾注（注释的注释）。

**Reply by PirateBook1** [reply]
这个情况很典型了

**Reply by ghost** [reply]
朗讀《全集》的時候經常會遇見的問題，問題就是是精讀還是略讀，是循環遞歸註釋閱讀還是跳過註釋閱讀的問題，遞歸起來甚至有可能有數層。
